secant/Dependencies/DerivationTool/DerivationToolTestKey.swift

```diff
@@ -6,58 +6,24 @@
 //
 
 import ComposableArchitecture
+import Foundation
```

Comment:
[non-blocking]
Just interested if Foundation is needed here or Xcode had a good day.